.github/workflows/dist.yml

```diff
@@ -6,6 +6,8 @@ on:
   push:
     branches:
       - main
+      - 'force_ci/all/**'    # For development, forcing all workflows to run.
+      - 'force_ci/dist/**'  # For debugging and/or only forcing this workflow.
```

Comment:
Slight misalignment of whitespace (if you're bothered about that).

Reply:
good catch, thanks!